libbpf-cargo/Cargo.toml

```diff
@@ -25,7 +25,7 @@ path = "src/lib.rs"
 
 [dependencies]
 anyhow = "1.0"
-cargo_metadata = "0.9"
+cargo_metadata = "0.12.3"
```

Comment:
Does it have to be exactly 0.12.3? I would usually prefer to leave it at 0.12 so we can pick up bug fix patches.

Reply:
Oh yeah that's fine too